android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java

```diff
@@ -313,9 +315,9 @@ private void createDatePicker() {
                 .build();
 
         picker.addOnPositiveButtonClickListener(e -> {
-            startDate = Instant.ofEpochMilli(e.first).atZone(ZoneId.systemDefault())
+            startDate = Instant.ofEpochMilli(e.first).atZone(ZoneId.of(TIME_ZONE))
                     .toLocalDate();
-            endDate = Instant.ofEpochMilli(e.second).atZone(ZoneId.systemDefault()).toLocalDate();
+            endDate = Instant.ofEpochMilli(e.second).atZone(ZoneId.of(TIME_ZONE)).toLocalDate();
 
             setDateCardTime();
```

Comment:
Here CodeCov mentions uncovered tests. I guess it is because the unit tests currently not working